src/Core/Services/IUserService.cs

```diff
@@ -65,18 +65,6 @@ public interface IUserService
     Task<bool> CheckPasswordAsync(User user, string password);
     Task<bool> CanAccessPremium(ITwoFactorProvidersUser user);
     Task<bool> HasPremiumFromOrganization(ITwoFactorProvidersUser user);
-    /// <summary>
-    /// Returns a list of user IDs and whether two factor is enabled for each user.
-    /// </summary>
-    /// <param name="userIds">The list of user IDs to check.</param>
-    Task<IEnumerable<(Guid userId, bool twoFactorIsEnabled)>> TwoFactorIsEnabledAsync(IEnumerable<Guid> userIds);
-    /// <summary>
-    /// Returns a list of users and whether two factor is enabled for each user.
-    /// </summary>
-    /// <param name="users">The list of users to check.</param>
-    /// <typeparam name="T">The type of user in the list. Must implement <see cref="ITwoFactorProvidersUser"/>.</typeparam>
-    /// <returns></returns>
-    Task<IEnumerable<(T user, bool twoFactorIsEnabled)>> TwoFactorIsEnabledAsync<T>(IEnumerable<T> users) where T : ITwoFactorProvidersUser;
     Task<bool> TwoFactorIsEnabledAsync(ITwoFactorProvidersUser user);
```

Comment:
Can you move the logic for this to the `TwoFactorIsEnabledQuery` as well? You can leave the property on the `UserService` so you don't have to change over all the calls, just add a TODO here for us to clean up later.

Reply:
I have marked it as obsolete